### PR TITLE
Add labels to fetch package

### LINKED ISSUE #425: add labels to @effection/fetch

### PR DESCRIPTION
**Pull request by jnicklas**
Closes #425

### CODE REVIEW

File: packages/fetch/src/fetch.ts [hunk 1]
```diff
@@ -29,26 +23,33 @@ export function fetch(info: RequestInfo, requestInit: RequestInit = {}): Fetch {
   }
 
   return {
+    name: `fetch \`${info}\``,
+    labels: {
+      expand: false
+    },
```
Comment (cowboyd) [inline packages/fetch/src/fetch.ts]: What about representing some of the `requestInit` inside the labels? It seems to me that parameters like `method` are extremely handy to have.

Comment (jnicklas) [reply packages/fetch/src/fetch.ts]: Good idea!

**Comment by jnicklas** [comment]
I added the request method in the labels and changed the naming scheme a bit, thinking we might adopt this naming scheme in other places (such as @effection/process) as well?

**Review by cowboyd — APPROVED** [review]
I like that naming scheme

**Pull request merged**